tests/console/slp.pm

```diff
@@ -27,18 +27,32 @@ sub run {
     my ($self) = @_;
     select_console 'root-console';
 
-    zypper_call 'in openslp openslp-server';
+    # Workaround: slpd requires its own user
+    # %pre statement in spec file of openslp-server creates dependency to daemon group provided by *system-user-daemon*
+    # /usr/sbin/useradd -r -g daemon -d /var/lib/empty -s /sbin/nologin -c "openslp daemon" openslp 2>/dev/null || :
+    # up to this point, JeOS doesn't require *system-user-daemon*
+    if (script_run 'getent group daemon') {
+        zypper_call 'in system-user-daemon';
```

Comment:
This only works for SLE 15 upwards - is that an issue?

Reply:
depends, if we want to run the test on sle12spX JeOS. I have checked that the group is present on the system. 
Or is there any MicroOS based on sle12?

Reply:
```
linux:~ # getent group daemon
daemon:x:2:
linux:~ # rpm -qa | grep kernel
kernel-default-base-4.12.14-120.1.x86_64
linux:~ # cat /etc/os-release
NAME="SLES"
VERSION="12-SP5"
VERSION_ID="12.5"
PRETTY_NAME="SUSE Linux Enterprise Server 12 SP5"
ID="sles"
ANSI_COLOR="0;32"
CPE_NAME="cpe:/o:suse:sles:12:sp5"
linux:~ # 
```

Reply:
Nope - so that should be fine